libs/api/metadata-converter/src/lib/gn4/gn4.field.mapper.ts

```diff
@@ -430,6 +430,7 @@ export class Gn4FieldMapper {
       /^OGC:WMS/.test(protocol) ||
       /^OGC:WFS/.test(protocol) ||
       /^OGC:WMTS/.test(protocol) ||
+      /^OGC:TMS/.test(protocol) ||
```

Comment:
we should only match on the "tms" word, since it can sometimes be encoded as `OSGeo:TMS` or event `TMS`

Reply:
Done